### PR TITLE
chore(docs): Robot stack release notes for 6.2.0

### PR DESCRIPTION
**Pull request by sfoster1**
Update app and api release notes for release 6.2.0.

### CODE REVIEW

File: api/release-notes.md [hunk 1]
```diff
@@ -6,19 +6,35 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
 
 ---
 
-# OT-2 Software Changes in 6.1.0
+# OT-3 Software Changes in 6.2.0
```
Comment (ecormany) [inline api/release-notes.md]:
```suggestion
# OT-2 Software Changes in 6.2.0
```
not yet 😇

**Review by ecormany — CHANGES_REQUESTED** [review]
1-byte change to be made, otherwise good!

**Review by ecormany — APPROVED** [review]
:shipit:

File: api/release-notes.md [hunk 2]
```diff
@@ -6,19 +6,35 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
...
+
+## Bug Fixes
+
+- Fixed a bug that could cause hardware modules to become unresponsive
+
+---
+
+## OT-2 Software Changes in 6.1.0
+
+Welcome to the v6.1.0 release of the OT-2 software! This release adds support for the Opentrons Heater-Shaker Module.
```
Comment (bndo) [inline api/release-notes.md:27]: should this be smoked?

Comment (bndo) [reply api/release-notes.md:27]: Ah, seems we've been keeping this language in since `5.0.2`so 🤷

**Pull request merged**